.gitignore

```diff
@@ -0,0 +1,2 @@
+.project
+.settings/
```

Comment:
This is fine, but FYI I think most of us are using our own `~/.gitignore` files.

Reply:
👍